src/main/java/spaceinvaders/group_22/ui/SpaceInvadersUI.java

```diff
@@ -15,17 +16,34 @@
  *
  */
 public class SpaceInvadersUI {
+	/**
+	 * Singleotn instance.
+	 */
+	private static SpaceInvadersUI instance;
```

Comment:
It is possible that someone mistakenly calls it in threads in the future, just a precaution to be safe.